src/ts/misc.ts

```diff
@@ -295,20 +295,30 @@ export async function reloadEntitiesShow(tag = ''): Promise<void | Response> {
   listenTrigger();
 }
 
-export async function reloadElement(elementId: string): Promise<void> {
-  if (!document.getElementById(elementId)) {
-    console.error(`Could not find element with id ${elementId} to reload!`);
+export async function reloadElements(elementIds: string[]|string): Promise<void> {
```

Comment:
why not having only `string[]` as param type? One element is then `reloadElements(['id'])`, and you can remove the if line 307.

Reply:
I simply like it more to avoid typing the additional square brackets. There are already 53 cases in the code base and there will probably be more calls to `reloadElements()` with only one id. So there are already 106 additional character compared to the ~~80~ 70 characters for the type check/conversion. We don't have to save the additional ~40 character, isn't it great 😜.
Sure there are two types and there is an additional condition but it is not a complex line of code.
Anyway, I will open an additional PR on this PR and you have the final word.